modules/electromagnetics/doc/content/source/kernels/VectorCoeffField.md

```diff
@@ -0,0 +1,13 @@
+# VectorCoeffField
```

Comment:
Coef is spelled with a single f in other objects

Reply:
as you want

Reply:
Thanks, holdover from a previous convention I followed. Missed this in the change to the framework convention.